README.md

```diff
@@ -81,13 +81,15 @@ If you want to integrate go-flutter with VSCode, read this [issue](https://githu
 
 ### Build standalone application
 
-To create a standalone debug build run this command:
```

Comment:
:warning: hover ONLY producing a JIT application.
People will assume `release build` are AOT build. 
Change this back to debug or something like `JIT release`
https://github.com/go-flutter-desktop/go-flutter/issues/259

Reply:
Well it's not a debug build either.. `flutter build bundle` does built a release version.
```
    --release                          Build a release version of your app
                                       (default mode).
```

Reply:
But indeed it's not AOT... I would call it "release (JIT mode)"